core-customize/hybris/bin/custom/sapcxtools/sapcommercetoolkit/testsrc/tools/sapcx/commerce/toolkit/testing/testdoubles/user/UserServiceFake.java

```diff
@@ -20,12 +16,77 @@
 import tools.sapcx.commerce.toolkit.testing.itemmodel.InMemoryModelFactory;
 
 public class UserServiceFake implements UserService {
+
+	public static Builder builder() {
+		return new Builder();
+	}
+
+	public static class Builder {
+		UserServiceFake fake = new UserServiceFake();
+
+		public Builder withCurrent(UserModel user) {
+			fake.current = user;
+			fake.users.put(user.getUid(), user);
+			return this;
+		}
+
+		public Builder withAdmin(EmployeeModel employeeModel) {
```

Comment:
We should not allow the admin user to be set from the outside. This is not supported anyway, as the admin user is always "admin". We may want to "deactivate" the account, but not change it.